tests/ground_destroy_test.cpp

```diff
@@ -25,7 +25,7 @@ TEST_CASE( "pavement_destroy", "[.]" )
     const ter_id flat_roof_id = ter_id( "t_flat_roof" );
```

Comment:
I can't attach this comment directly on it, but there's a code comment up on line 22 that can be deleted.